app/bundles/MessengerBundle/Tests/MessageHandler/EmailHitNotificationHandlerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Mautic\MessengerBundle\Tests\MessageHandler;
+
+use Mautic\EmailBundle\Model\EmailModel;
+use Mautic\MessengerBundle\Message\EmailHitNotification;
+use Mautic\MessengerBundle\MessageHandler\EmailHitNotificationHandler;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class EmailHitNotificationHandlerTest extends TestCase
+{
+    public function testInvoke(): void
+    {
+        $hitId   = sha1((string) rand(0, 1000000));
+        $request = new Request();
+        $request->query->set('testMe', 'I am here');
+
+        /** @var MockObject|EmailModel $emailModelMock */
+        $emailModelMock = $this->createMock(EmailModel::class);
+        $emailModelMock
+            ->expects($this->exactly(1))
+            ->method('hitEmail')
+            ->with($hitId, $request)
+        ;
```

Comment:
```suggestion
            ->with($hitId, $request);
```
Please follow the code style standard that is in other files. This applies to many files in this PR.